src/boot/store.js

```diff
@@ -283,6 +283,14 @@ const migrations: {| [string]: (GlobalState) => GlobalState |} = {
     accounts: state.accounts.filter(a => a.email !== ''),
   }),
 
+  '28': state => ({
```

Comment:
Should get a quick explanation, like the other migrations above. (Chris already took care of this in 74ae528f0.)